npm/oxlint/bin/oxlint

```diff
@@ -91,25 +92,25 @@ const PLATFORMS = {
 };
 
 let binPath = (
-  PLATFORMS && 
-  PLATFORMS[platform] && 
-  PLATFORMS[platform][arch] && 
+  PLATFORMS &&
+  PLATFORMS[platform] &&
+  PLATFORMS[platform][arch] &&
   PLATFORMS[platform][arch][isMusl() ? "musl" : "gnu"]
 ) || null;
 
 if (binPath) {
+  const newEnvObject = Object.assign({}, env, {
```

Comment:
We can inline this.

Reply:
Done too. (removed the `newEnvObject` variable)